src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -18,6 +18,45 @@ internal static partial class RoslynExtensions {
 			"System.IO.Abstractions.IFileSystem"
 		);
 
+		/// <summary>

```

Comment:
These might be better in their own extension methods class.  Everything to do with immutable containers.